packages/nodejs/src/client.ts

```diff
@@ -28,6 +28,12 @@ export class BaseClient implements Client {
   #tracer: Tracer = new BaseTracer()
   #metrics: Metrics = new BaseMetrics()
 
+  /**
+   * Global accessors to Client and Config
+   */
+  static client: Client = global.__APPSIGNAL__
```

Comment:
If I call `Appsignal.client` in a test app it returns `undefined`. Could be that it doesn't update when we overwrite the value?

```
const { Appsignal } = require("@appsignal/nodejs");
const appsignal = new Appsignal({
  "active": true,
});

console.log("!!! global", Appsignal.client);
// => undefined
```

Reply:
I think it has to be a getter, otherwise it'll grab the value of `global.__APPSIGNAL__` when the class is defined.

Reply:
Yep, it gets the value when the class is defined but it is overwritten every time the config is initialized on `storeInGlobal()` and it worked on my tests. I'll do a couple more, if it doesn't work we should go with the static functions.

Reply:
The results are not consistent. I would go with static functions instead to avoid future bugs.